public/index.html

```diff
@@ -54,6 +54,17 @@
 			</figure>
 		</section>
 
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		<footer>
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+			<ul>
+				<li><a href="#">Home</a></li>
+				<li><a id="github">⚙ Contribute to this page</a></li>
+			</ul>
+		</footer>
+		<script type="text/javascript">
```

Comment:
Is it necessary to have a script here when there is only one page ?

Reply:
Yes in order to extend this functionality to all pages easily at the end of this sprint.